action.yml

```diff
@@ -28,7 +32,7 @@ runs:
         DITTO_API_KEY: ${{ inputs.ditto-api-key }}
       shell: bash
       run: |
-        npx ditto-cli pull -m githubActionRequest:true
+        cd ${{ inputs.root-dir }} && npx ditto-cli pull -m githubActionRequest:true
```

Comment:
I wonder if we should instead simply utilize [the env var](https://github.com/dittowords/cli/pull/26) that the CLI supports for specifying the output directory here?